tools/nni_cmd/nnictl_utils.py

```diff
@@ -433,3 +436,38 @@ def monitor_experiment(args):
         except Exception as exception:
             print_error(exception)
             exit(1)
+
+
+def parse_trial_data(trial_data):
+    """output: Dict"""
+    hparam = json.loads(trial_data['hyperParameters'][0])['parameters']
```

Comment:
how to deal with multi-phase enabled experiments?

Reply:
it's not taken into account yet, for intermediate result variate length sequence. I don't know how to convert to a fixed csv column yet.